gen/src/generate/idna/mapping.rs

```diff
@@ -65,11 +65,11 @@ impl IdnaMapping {
                 },
             )?;
             if let Some(ref chars) = line.mapping {
-                write!(file, "(&[")?;
+                write!(file, "(\"")?;
                 for char in chars.iter() {
-                    write!(file, "'{}',", char.escape_unicode())?;
+                    write!(file, "{}", char.escape_unicode())?;
```

Comment:
Maybe add a TODO comment on the for loop to switch to `str::escape_unicode()` when it's stable?